<?xml version="1.0"?>
<launch>
    <arg name="settings" default="/home/sahib/Downloads/BagFiles/magna.yaml" />
    <arg name="vocabulary" default="/home/sahib/Documents/ORBSlam/Vocabulary/ORBvoc.txt" />
    <arg name="rosbag" default="/home/sahib/Downloads/BagFiles/test_taiwan.bag" />

    <param name="VOCAB_FILE" value="$(arg vocabulary)" />
    <param name="SETTINGS_FILE" value="$(arg settings)" />

    <!-- Launch the ORB Slam Node and the rosbag player -->
    <node pkg="orb_slam" name="mono_slam" type="mono_slam" output="screen" >
        <remap from="/camera/image_raw" to="/VIDImg_Left" />
    </node>
    <node pkg="rosbag" type="play" name="rosbag_player" args="-l $(arg rosbag)" />
    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find orb_slam)/rviz/viewer.rviz" /> -->

</launch>